statsmodels/base/_penalized.py

```diff
@@ -98,18 +114,20 @@ def score(self, params, pen_weight=None, **kwds):
 
         sc = super(PenalizedMixin, self).score(params, **kwds)
         if pen_weight != 0:
-            sc -= pen_weight * self.penal.grad(params)
+            scale = self._handle_scale(params, **kwds)
+            sc -= 1/scale * pen_weight * self.penal.deriv(params)
 
         return sc
 
-    def score_obs(self, params, pen_weight=None, **kwargs):
+    def score_obs(self, params, pen_weight=None, **kwds):
         if pen_weight is None:
             pen_weight = self.pen_weight
 
-        sc = super(PenalizedMixin, self).score_obs(params, **kwargs)
+        sc = super(PenalizedMixin, self).score_obs(params, **kwds)
```

Comment:
why this change?

Reply:
OK.  If this could be made consistent across the code that'd be swell, but clearly not necessary for this PR.